internal/olm/operator/internal/registry_pod.go

```diff
@@ -51,6 +49,11 @@ const (
 	defaultGRPCPort          = 50051
 )
 
+var (
+	// Internal error
+	errPodNotInit = errors.New("internal error: RegistryPod not initialized")
```

Comment:
It is used just in one place in the code so has no need to be global and be done where the error should be returned.